tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -1109,8 +1113,35 @@ test.describe( 'Nested Tabs tests @nested-tabs', () => {
 		await page.locator( '.eicon-video-camera' ).first().click();
 		await page.locator( '.elementor-control-background_video_link input' ).fill( videoUrl );
 
+		await editor.selectElement( contentContainerTwoId );
+		await editor.activatePanelTab( 'style' );
+		await page.locator( '.eicon-video-camera' ).first().click();
+		await page.locator( '.elementor-control-background_video_link input' ).fill( videoUrl );
```

Comment:
```
await page.locator( '.eicon-video-camera' ).first().click();
await page.locator( '.elementor-control-background_video_link input' ).fill( videoUrl );
```
Suggestion.
I see that the lines above are used in several places. 
We can extract it to a local function.

Reply:
Done